src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,75 @@
+const { errorHandler } = require('../helpers/errorHandler');
+const {
+  expensesService,
+  initExpensesService,
+} = require('../services/expenses.service');
+const { STATUS_CODES } = require('../variables/variables');
+const express = require('express');
+
+const expensesController = () => {
+  initExpensesService();
+
+  const expensesRoutes = express.Router();
+
+  expensesRoutes.get('/', (req, res) => {
```

Comment:
Don't create routes in controllers, separate them as I've mentioned in this comment https://github.com/mate-academy/node_accounting-app-v2/pull/597#discussion_r1607006482